packages/core/tests/site/index.html

```diff
@@ -170,7 +170,7 @@ <h2>function: AU-pow</h2>
 
 	<code class="au-pow1">5<sup>3</sup> = 125:</code><br>
 	<code class="au-pow2">4<sup>2</sup> = 16:</code><br>
-	<code class="au-pow3">4<sup>-2</sup> = 0.0625:</code><br>
+	<!-- <code class="au-pow3">4<sup>-2</sup> = 0.0625:</code><br> -->
```

Comment:
was that because we disabled rounding?

Reply:
Yeah 👍

Reply:
Oh this line of code was always commented out. It errors as:

```
@if $exponent < 0 {
	@error "I can see dead people; the AU-pow function only takes positive numbers as the second argument.";
}
```